tiatoolbox/utils/misc.py

```diff
@@ -712,6 +712,50 @@ def string_to_tuple(in_str):
     return tuple(substring.strip() for substring in in_str.split(","))
 
 
+def prepare_file_dir_cli(img_input, output_path, file_types, mode, sub_dirname):
+    """Prepares cli for running code on multiple files or a directory.
+
+    Checks for existing directories to run tests.
+    Converts file path to list of file paths or
+    creates list of file paths if input is a directory.
```

Comment:
I am still a bit unclear what this is for? Is it for testing only? If so, then it should be under tests rather than in the main toolbox IMO.

Reply:
avoiding duplicate codes for multiple cli inputs. Will replace it in future with a class definition.